.github/workflows/stockfish.yml

```diff
@@ -35,18 +40,22 @@ jobs:
               os: ubuntu-20.04,
               compiler: clang++,
               comp: clang,
-              run_expensive_tests: false,
-              run_32bit_tests: true,
               run_64bit_tests: true,
               shell: 'bash {0}'
             }
+          - {
+              name: "Ubuntu 20.04 NDK armv8",
+              os: ubuntu-20.04,
+              compiler: aarch64-linux-android21-clang++,
```

Comment:
is that getting automatically installed? Basically, where does it requires the Android NKD to be available?

Reply:
NDK already installed on Ubuntu 20.04 Github Virtual Environment
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#android

Reply:
ah, nice, that's really a fully featured image, not stock ubuntu.

Reply:
PGO for armv8 in test
https://github.com/ppigazzini/Stockfish/runs/5096366759?check_suite_focus=true